cmd/platform/export.go

```diff
@@ -69,7 +77,19 @@ func exportRunE(cmd *cobra.Command, args []string) error {
 
 func initGeneralExportFlags(cmd *cobra.Command) {
 	cmd.Flags().AddFlag(options.PlatformExportExportFormatOption.Flag)
+	// auto-completion
+	err := cmd.RegisterFlagCompletionFunc("format", autocompletion_export_flags.Format)
+	if err != nil {
+		returnExportSystemError(err)
```

Comment:
Probably better to inline this instead, with the added benefit of the error message being more descriptive of the flag in question